libvcx/src/api_lib/api_c/schema.rs

```diff
@@ -387,45 +387,11 @@ pub extern fn vcx_schema_get_attributes(command_handle: CommandHandle,
 ///         ]
 ///     }
 #[no_mangle]
-pub extern fn vcx_schema_get_payment_txn(command_handle: CommandHandle,
-                                         handle: u32,
-                                         cb: Option<extern fn(xcommand_handle: CommandHandle, err: u32, txn: *const c_char)>) -> u32 {
+pub extern fn vcx_schema_get_payment_txn(_command_handle: CommandHandle,
+                                         _handle: u32,
+                                         _cb: Option<extern fn(xcommand_handle: CommandHandle, err: u32, txn: *const c_char)>) -> u32 {
     info!("vcx_schema_get_payment_txn >>>");
-
-    check_useful_c_callback!(cb, VcxErrorKind::InvalidOption);
-
-    trace!("vcx_schema_get_payment_txn(command_handle: {})", command_handle);
-
-    execute(move || {
-        match schema::get_payment_txn(handle) {
-            Ok(x) => {
-                match serde_json::to_string(&x) {
-                    Ok(x) => {
-                        trace!("vcx_schema_get_payment_txn_cb(command_handle: {}, rc: {}, : {}), source_id: {:?}",
-                               command_handle, error::SUCCESS.message, x, schema::get_source_id(handle).unwrap_or_default());
-
-                        let msg = CStringUtils::string_to_cstring(x);
-                        cb(command_handle, 0, msg.as_ptr());
-                    }
-                    Err(e) => {
-                        let err = VcxError::from_msg(VcxErrorKind::InvalidJson, format!("Cannot serialize payment txn: {}", e));
-                        error!("vcx_schema_get_payment_txn_cb(command_handle: {}, rc: {}, txn: {}), source_id: {:?}",
-                               command_handle, err, "null", schema::get_source_id(handle).unwrap_or_default());
-                        cb(command_handle, err.into(), ptr::null_mut());
-                    }
-                }
-            }
-            Err(x) => {
-                error!("vcx_schema_get_payment_txn_cb(command_handle: {}, rc: {}, txn: {}), source_id: {:?}",
-                       command_handle, x, "null", schema::get_source_id(handle).unwrap_or_default());
-                cb(command_handle, x.into(), ptr::null());
-            }
-        };
-
-        Ok(())
-    });
-
-    error::SUCCESS.code_num
+    return VcxError::from_msg(VcxErrorKind::ActionNotSupported, format!("Payment api not supported.")).into()
```

Comment:
We won't break anything if we remove this, I would expect?

Reply:
Nope this should be fine

Reply:
I meant removing the function (`vcx_schema_get_payment_txn`) :)

Reply:
then I would have to remove from all the wrappers as well, otherwise we'd likely see `undefined symbol` type of errors. Let's do that in separate step.

Reply:
removed all unsupported functions from FFI and wrappers